variables.tf

```diff
@@ -75,6 +75,12 @@ DESCRIPTION
   }
 }
 
+variable "create_compute_instance" {
+  type        = bool
+  default     = true
```

Comment:
default to false

Reply:
Good point!  Default value is set to false.